crates/controller/src/selection/bookkeeping.rs

```diff
@@ -164,15 +164,13 @@ fn update_selection(mut events: EventReader<SelectEvent>, selector_builder: Sele
 
 fn selected_system(
     mut events: EventReader<SelectedEvent>,
-    mut markers: Query<&mut CircleMarker>,
+    mut markers: Query<&mut MarkerVisibility>,
     mut bars: EventWriter<UpdateBarVisibilityEvent>,
     mut poles: EventWriter<UpdatePoleVisibilityEvent>,
 ) {
     for event in events.iter() {
-        if let Ok(mut marker) = markers.get_mut(event.0) {
-            marker
-                .visibility_mut()
-                .update_visible(SELECTION_BAR_ID, true);
+        if let Ok(mut visibility) = markers.get_mut(event.0) {
```

Comment:
What do we do if not Ok? Or is this something that should never happen?

Reply:
This is because `SelectedEvent` for a particular entity might have been sent during an earlier scheduling base set. Therefore, other systems might get a chance to despawn the entity.